app/templates/dataset.html

```diff
@@ -97,21 +103,23 @@ <h2><span style="color:red;">CONP Portal</span> | {{ data.title }} </h2>
   {% endif %}
 </div>
 
+{% if not showDownloadButton %}
```

Comment:
@liamocn I think we can still show "alternate" download instruction even when the download button is there.
@cmadjar ?

Reply:
I'm open to that, if that's the decision. Maybe just change the section heading if there's a download button instead?

Reply:
I agree with Xavier, we should keep the instructions as alternate download instructions.

Question: if there is an error when clicking on the download button, does the user get a message? For example, if there is a `datalad get` error?

Reply:
Hi @cmadjar @xlecours . I've made these changes in a PR mentioned below. The download instructions are back and there is both a progress view and an error view to provide feedback to the user.